### DISCUSSION TITLE
sBTC Design - Voting

### DISCUSSION THREAD

**Discussion by AshtonStephens** [stacks-archive/sbtc-developer-release#356] [Ideas]
This is a design discussion around the stacker / signer voting required for sBTC. The key notes are:
1. What are people voting on
2. What triggers a voting round
3. Any other unknowns

**Comment by AshtonStephens** [comment]
Key voting items
- Minimum deposit value
- Max fee
- Threshold percentage %
- DKG wallet

**Reply by AshtonStephens** [reply]
Minimum deposit value will partially handle the fact that we could get a DDOS of challenge transactions

**Reply by andrerserrano** [reply]
@AshtonStephens do we have clarity on which of these items require a vote and how it will be handled? I reviewed the doc below, but curious about final decisions.

**Comment by AshtonStephens** [comment]
Under investigation by @MarvinJanssen

**Reply by AshtonStephens** [reply]
[sBTC Voting Mechanism](https://docs.google.com/document/d/1Tra7LhVQjMstm5h9iYVekdbdsn29iUjfEmlbQEsNFRU/edit?usp=sharing)
